src/sonic-yang-models/yang-events/sonic-events-dhcp_relay.yang

```diff
@@ -0,0 +1,75 @@
+module sonic-events-dhcp-relay {
+    namespace "http://github.com/sonic-net/sonic-events-dhcp-relay";
+    yang-version 1.1;
+
+    import openconfig-alarm-types {
+        prefix oc-alarm-types;
+    }
+
+    import sonic-events-common {
+        prefix evtcmn;
+    }
+
+    revision 2022-12-01 {
+        description "dhcp-relay alert events.";
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC dhcp-relay events";
+
+    container dhcp-relay-discard {
```

Comment:
Top-level container

Reply:
YANG review is moved to different [PR #11404 ](https://github.com/Azure/sonic-buildimage/pull/11404)